examples/swe_agent/swe_agent_prompts.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=C0301
+"""The SWE-agent relay heavily on it's prompts.
+This file contains the neccessary prompts for the SWE-agent.
+Some prompts are taken and modified from the original SWE-agent repo
+or the SWE-agent implementation from Open-Devin.
+"""
+
+WINDOW = 100
+
+
+def get_system_prompt(command_prompt: str) -> str:
```

Comment:
Why don't we place the system prompt within the swe_agent.py file?

Reply:
Too many prompts here so i move most of them in `swe_agent_prompts.py`